client/my-sites/checkout/concierge-session-nudge/index.jsx

```diff
@@ -0,0 +1,75 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import QuerySites from 'components/data/query-sites';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import CompactCard from 'components/card/compact';
+import Button from 'components/button';
+
+export class ConciergeSessionNudge extends React.Component {
+	static propTypes = {
+		receiptId: PropTypes.number.isRequired,
+		selectedSiteId: PropTypes.number.isRequired,
+	};
+
+	render() {
+		const { receiptId, selectedSiteId, translate } = this.props;
+		const title = translate( 'My Sites > Checkout > Expert Session' );
```

Comment:
Thanks for catching that @ramonjd.
I used the format I did based on some "standard" that I thought I read in the docs somewhere, but I can't find it now.
I've updated it to be more inline with your example (and other page) and removed the translate call for now, until the page is finalized.

I'm not finding how I'd be able to set the title just to "Expert Session" and have it automatically pull in the other elements (eg. "Checkout < ...), if that's what you mean. Do you happen to have any examples of where that's being done.

Reply:
I looked into this a bit further and see that that `<DocumentHead />` is doing all the work anyway: it calls `setTitle`, then formats the title by calling `getDocumentHeadFormattedTitle`, which appends any site (if it's found) and then `- WordPress.com`.

Passing a `title` prop (of `translate( 'Expert Session' )` or whatever) I think is all you need to do to get it to look like the thank you page:

<img width="252" alt="screen shot 2018-11-01 at 11 32 05 am" src="https://user-images.githubusercontent.com/6458278/47826163-35c47580-ddca-11e8-8c98-37e286c9842b.png">

On the other hand, since you're in the middle of the checkout process, you might want to show the site title, e.g., `Expert Session ‹ A sweet site — WordPress.com`. `getDocumentHeadFormattedTitle` add this when it's [part of a section](https://github.com/Automattic/wp-calypso/blob/fa32cd0/client/state/ui/selectors.js#L92).

To force x to be part of the checkout section, I just swapped the route params:
`'/checkout/add-expert-session/:site/:receiptId?',`

Then you can hide the side bar in the controller method `conciergeSessionNudge`:
```
export function conciergeSessionNudge( context, next ) {
	context.store.dispatch( setSection( null, { hasSidebar: false } ) );
     ...
```

Sorry, it's a lot of info for the early stages of the component... I fell down the rabbit hole!!!

Reply:
Thanks for all the details @ramonjd - that's helpful!
I think I'll deploy it as is but might end up revisiting this in a future iteration.